views/index.ejs

```diff
@@ -5,11 +5,11 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <meta http-equiv="X-UA-Compatible" content="ie=edge">
   <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.4.1/css/bootstrap.min.css" integrity="sha384-Vkoo8x4CGsO3+Hhxv8T/Q5PaXtkKtu6ug5TOeNV6gBiFeWPGFN9MuhOf23Q9Ifjh" crossorigin="anonymous">
-  <title>Document</title>
+  <title>URL Shortener</title>
 </head>
 <body>
   <div class="container">
-    <h1>URL Shrinker</h1>
+    <h1>URL Shortener</h1>
```

Comment:
I do not like to change the files from the course after creating the video so could you remove the changes to the index.ejs file.

Reply:
I understand. I thinked about this when I made the modifications. Sorry about grammar, I am brazilian and my english is not good yet. I liked the project and wanted to expand it. Maybe you should pass to your Git Hub account and separate from that account where you post the course codes. This way, you will also allow your subscribers to submit ideas for the projects you post on the channel. It's just an idea, if you think it's valid send me your Git Hub where you will post these projects as Open Source, I like to contribute. Anyway, congratulations to your videos.